src/eos-updater-fetch.c

```diff
@@ -93,6 +95,21 @@ repo_pull (OstreeRepo *self,
     g_variant_builder_add (&builder, "{s@v}", "override-url",
                            g_variant_new_variant (g_variant_new_string (url_override)));
 
+  /* Get gpg-verify-summary setting, defaulting to TRUE unless it’s explicitly
+   * disabled. */
+  if (!ostree_repo_get_remote_boolean_option (self, remote_name,
+                                              "gpg-verify-summary", TRUE,
+                                              &gpg_verify_summary,
+                                              &child_error))
+    {
+      g_warning ("Failed to read gpg-verify-summary option from repository: %s",
+                 child_error->message);
+      gpg_verify_summary = TRUE;
+    }
```

Comment:
`ostree_repo_get_remote_boolean_option` won't return an error on the option not being, so I feel like this might mask real errors in the repo. But probably safest to keep going instead of bombing here.